language/documentation/book/src/structs-and-resources.md

```diff
@@ -288,104 +290,102 @@ Note that structs do not have visibility modifiers (e.g., `public` or `private`)
 
 ## Ownership
 
-By default, structs can be freely copied and silently dropped.
+As mentioned above in [Defining Structs](#defining-structs), structs are by default linear and ephemeral. This means they cannot be copied or dropped. This property can be very useful when modeling real world resources like money, as you do not want money to be duplicated or get lost in circulation.
 
-```rust=
+```move=
 address 0x2 {
 module M {
     struct Foo { x: u64 }
 
-    public fun run() {
-        let foo = Foo { x: 100 };
-        let foo_copy = copy foo;
-        // ^ this code copies foo, whereas `let x = foo` or
-        // `let x = move foo` both move foo
-
-        let x = foo.x;            // x = 100
-        let x_copy = foo_copy.x;  // x = 100
-
-        // both foo and foo_copy are implicitly discarded when the function returns
-    }
-}
-}
-```
-
-Resource structs on the other hand, cannot be copied or dropped silently. This property can be very useful when modeling real world resources like money, as you do not want money to be duplicated or get lost in circulation.
-
-```rust=
-address 0x2 {
-module M {
-    resource struct Foo { x: u64 }
-
     public fun copying_resource() {
         let foo = Foo { x: 100 };
-        let foo_copy = copy foo; // error! resources cannot be copied
+        let foo_copy = copy foo; // error! cannot be copy without the 'copy' ability
         let foo_ref = &foo;
-        let another_copy = *foo_ref // error! resources cannot be copied
+        let another_copy = *foo_ref /// error! cannot be copy without the 'copy' ability
```

Comment:
Why 3 slashes before the comment? This comment doesn't seem quite right, either. It seems awkward to say "cannot be copy" (also above), and in this case, the copy is only implied by the dereference. The actual error message is: "Invalid dereference. Dereference requires the 'copy' ability". The last part of that message would fit well here.

Reply:
Cleaned up the surrounding ones too